decidim-core/app/views/layouts/decidim/_head.html.erb

```diff
@@ -27,6 +27,7 @@
 <link rel="manifest" href="/manifest.webmanifest">
 
 <%= prepend_javascript_pack_tag "decidim_core", defer: false %>
+<%= append_stylesheet_pack_tag "decidim_overrides", media: "all" %>
 <%= stylesheet_pack_tag "decidim_core", media: "all" %>
```

Comment:
Appends should be after the appended file IMO (if it's possible and it doesn't break the feature of course) 
```suggestion
<%= stylesheet_pack_tag "decidim_core", media: "all" %>
<%= append_stylesheet_pack_tag "decidim_overrides", media: "all" %>
```

Reply:
Ok, a bit counter intuitive at least for me, but if as I thought that this may be the case